docs/source/tutorial/namespace_pipelines.md

```diff
@@ -153,7 +170,6 @@ In this section we want to add some namespaces in the modelling component of the
             pipe=pipeline_instance,
             inputs="model_input_table",
             namespace="candidate_modelling_pipeline",
-            parameters={"params:model_options": "params:model_options_experimental"},
```

Comment:
I have removed the `parameters` here and below to reflect the `spaceflights` starter.